test/Libraries/RevitNodesTests/Elements/Views/SheetTests.cs

```diff
@@ -125,5 +125,44 @@ public void ByNameNumberTitleBlockAndView_BadArgs()
             Assert.Throws(typeof(ArgumentNullException), () => Sheet.ByNameNumberTitleBlockAndView(sheetName, sheetNumber, titleBlock, null));
         }
 
+        [Test]
+        [TestModel(@".\Empty.rvt")]
+        public void ByNameNumberTitleBlock_ValidArgs()
+        {
+            ElementBinder.IsEnabled = false;
+
+            var famSymName = "E1 30x42 Horizontal";
+            var famName = "E1 30 x 42 Horizontal";
+            var titleBlock = FamilyType.ByFamilyAndName(Family.ByName(famName), famSymName);
+
+            var famTyp = FamilyType.ByName("Kousa Dogwood - 10'");
+
+            var sheetName = "Poodle";
+            var sheetNumber = "A1";
+
+            var ele = Sheet.ByNameNumberTitleBlock(sheetName, sheetNumber, titleBlock);
+
+            Assert.NotNull(ele);
```

Comment:
we should also check that the values are the expected ones for
- name
- number
- titleblock family type